src/api/graphql.generated.tsx

```diff
@@ -62,11 +62,6 @@ export interface AttachUserToSecurityGroupInput {
   request: AttachUserToSecurityGroup;
 }
 
-export interface BaseNodeConsistencyInput {
```

Comment:
Your server code is behind causing this generated schema to regress. Please pull the latest on API and re-gen this.